tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py

```diff
@@ -336,14 +340,18 @@ def set_estimator_and_feature_spec(self, estimator, feature_spec):
     Returns:
       self, in order to enabled method chaining.
     """
+    # If custom function is set, remove it before setting estimator
+    self.delete('custom_predict_fn')
+    self.delete('compare_custom_predict_fn')
```

Comment:
i think this should jsut delete the custom_predict_fn, not the compare one

Reply:
done, makes sense if you want to compare an estimator to a predict function.